pkg/version/version.go

```diff
@@ -38,8 +38,8 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                  = "0.12.0"
-	PrevReleaseVersion         = "0.11.0"
+	Promscale                  = "0.13.0-dev.0"
```

Comment:
I guess that the `.0` after `-dev` is no longer necessary, since Promscale isn't dealing with SQL migrations anymore.